src/test/java/com/uber/profiling/ArgumentsTest.java

```diff
@@ -57,6 +59,14 @@ public void emptyArgumentValue() {
         Arguments.parseArgs("reporter=,durationProfiling=,metricInterval=,appIdRegex=,");
     }
 
+    @Test
+    public void noop() {
+        Arguments arguments = Arguments.parseArgs("durationProfiling=a.bc.foo,noop=true,durationProfiling=ab.c.d.test");
+        Assert.assertEquals(2, arguments.getRawArgValues().size());
+        Assert.assertTrue(arguments.isNoop());
+        Assert.assertEquals(2, arguments.getDurationProfiling().size());
+    }
```

Comment:
do we need to have a test to verify when noop is set, no profiling would be executed?

Reply:
Good suggestion! Will add a test.